src/shogun/neuralnets/StanNeuralLogisticLayer.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Elfarouk, Khaled Nasr
+ */
+
+#ifndef __STANNEURALLOGISTICLAYER_H__
+#define __STANNEURALLOGISTICLAYER_H__
+
+#include <shogun/neuralnets/StanNeuralLinearLayer.h>
+
+namespace shogun
+{
+/** @brief Neural layer with linear neurons, with a [logistic activation
+ * function](http://en.wikipedia.org/wiki/Logistic_function). can be used as a
+ * hidden layer or an output layer.
+ *
+ * When used as an output layer, a
+ * [squared error measure](http://en.wikipedia.org/wiki/Mean_squared_error) is
+ * used
+ */
+class StanNeuralLogisticLayer : public StanNeuralLinearLayer
```

Comment:
Perhaps it is only a naming issue, but saying that a LogisticLayer is a LinearLayer sounds odd.

Reply:
Any plans to fix?